useraccount/internal/adapters/framework/driver/grpc/proto/rpc.go

```diff
@@ -0,0 +1,21 @@
+package rpc
```

Comment:
same for this file. This should be in grpc folder.

Reply:
That's how it is in the video ?